libs/gf/msw/util-msw/src/lib/handlers.ts

```diff
@@ -14,38 +14,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {
-  DefaultBodyType,
-  GraphQLHandler,
-  GraphQLRequest,
-  MockedRequest,
-  RequestHandler,
-  RestHandler,
-} from 'msw';
+import { GraphQLHandler, RequestHandler, HttpHandler } from 'msw';
 
-export type mocks = ((
-  apiBase: string
-) => (RestHandler<MockedRequest<DefaultBodyType>> | GraphQLHandler<GraphQLRequest<never>>)[])[];
+export type mocks = ((apiBase: string) => (HttpHandler | GraphQLHandler)[])[];
 
 export function handlers(apiBase: string, mocks: mocks): RequestHandler[] {
   return mocks.map((mock) => mock(apiBase)).flat();
```

Comment:
nitpick: bumping the `ES<X>` version in `tsconfig` can lead to issues with Jest/Jest DOM/whatever failing because it doesn't support a newer version of EcmaScript, and an older one doesn't support the `flat` method. One way to solve this is to rewrite the logic using `reduce`, for example.

It's not necessary now since everything is working, but let's keep it in mind in case we get a `flat` error again.